extensions/amp-bind/0.1/bind-expression.js

```diff
@@ -257,6 +267,15 @@ export class BindExpression {
   }
 
   /**
+   * @param {*} target
+   * @param {*} member
+   */
+  memberAccessWarning_(target, member) {
+    user().warn(TAG, `Cannot read property ${member} of ${target}; ` +
```

Comment:
Do we show this without a custom log level?
I'm not sure, but we probably should with `#development=1` in the URL